### PR TITLE
[Junos]: smoke tests added

### PR DESCRIPTION
**Pull request by rohitthakur2590**
Depends-On: https://github.com/ansible-collections/junipernetworks.junos/pull/62
Signed-off-by: Rohit Thakur <rohitthakur2590@outlook.com>

##### ISSUE TYPE

 
- Feature Pull Request
 

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: tests/integration/targets/junos_smoke/tests/netconf/common_utils.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  when: result['ansible_facts']['ansible_net_model']  is search("vqfx*")
+
+- name: Check intent arguments
+  junos_interface:
+    name: "{{ intf_name }}"
+    state: up
+    tx_rate: ge(0)
+    rx_rate: le(0)
+    provider: "{{ netconf }}"
+  register: result
```
Comment (ganeshrn) [inline tests/integration/targets/junos_smoke/tests/netconf/common_utils.yaml:33]: junos_interface module is deprecated. Please add a test for junos_interfaces module instead.

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Pull request merged**